KTL.js

```diff
@@ -5619,7 +5621,28 @@ function Ktl($, appInfo) {
                 }
             }
         }
-
+        function makeTableHeaderSticky(viewId, keywords, data) {
+            const kw = '_sth';
+            let numOfRecords = 10;
+            let viewHeight = 800;
+            if (keywords[kw] && keywords[kw][0]?.params && keywords[kw][0]?.params[0].length) {
+                numOfRecords = keywords[kw][0].params[0][0] ? keywords[kw][0].params[0][0] : numOfRecords;
+                if (keywords[kw][0].params[0][1]) viewHeight = keywords[kw][0].params[0][1];
```

Comment:
Why a ternary on one line and not on the second? Mixmatch of code style.
You could have also coded this as `viewHeight = keywords[kw][0].params[0][1] || viewHeight` and it would have done the same. There is no better option other than being consistent.

Reply:
Ok Thank you.  I was struggling with the keyword having optional parameters if I used _sth instead of _sth=6,1000 params wasn't there to check and threw an error.  I will rework the code.